### PR TITLE
chore: tweak lint config to allow for `eslint --fix` to fix import order

### PR DESCRIPTION
**Pull request by filmaj**
Small maintainer QoL fix: tweaked the eslint config to use a combination of ESLint's stock `sort-imports` rule and `eslint-plugin-import`'s `import/order` rule to allow for `eslint --fix` to fix import ordering. This was driving me a bit crazy _eyetwitch_ and I found [this comment](https://github.com/import-js/eslint-plugin-import/issues/2521#issuecomment-1216630234) that showed off using the two rules in combo.

Also ensured all the `lint` npm run scripts use the `--fix` rule across all monorepo packages.

The only change worth reviewing is within `lint-configs/` - all other file changes are simply style fixes.

### CODE REVIEW

File: packages/cli-test/src/cli/shell.spec.ts [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
-import stream from 'stream';
 import EventEmitter from 'events';
+import stream from 'stream';
+
+import { assert } from 'chai';
+import sinon from 'sinon';
+
 import { shell } from './shell';
+
 import type { ShellProcess } from '../utils/types';
```
Comment (zimeg) [inline packages/cli-test/src/cli/shell.spec.ts:10]: 😱 Big fan of splitting stdlib from dependencies from local from types if that's what this rule is requiring!

File: packages/cli-test/src/utils/logger.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { createLogger, format, transports, Logger } from 'winston';
+import { Logger, createLogger, format, transports } from 'winston';
```
Comment (zimeg) [inline packages/cli-test/src/utils/logger.ts:1]: Uppercase alphabetization is a big winston 🏆

File: packages/web-api/src/methods.ts [hunk 3]
```diff
@@ -222,272 +488,8 @@ import type {
...
-  AdminWorkflowsSearchResponse,
-  AdminWorkflowsUnpublishResponse,
-  AdminWorkflowsCollaboratorsAddResponse,
-  AdminWorkflowsCollaboratorsRemoveResponse,
-  AdminWorkflowsPermissionsLookupResponse,
-  AdminAppsConfigLookupResponse,
-  AdminAppsConfigSetResponse,
 } from './types/response/index';
 // Request types
-import type {
```
Comment (zimeg) [inline packages/web-api/src/methods.ts:248]: 🗒️ Might be nice to also swap 🪓 the `request` and `response` comments for these moving types!

File: packages/web-api/src/types/request/chat.ts [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
   TokenOverridable,
 } from './common';
-import { OptionalArgument } from '../helpers';
+import type {
+  Block, // TODO: these will be combined into one in a new types release
+  KnownBlock,
+  LinkUnfurls,
+  MessageAttachment,
+  MessageMetadata,
+} from '@slack/types';
```
Comment (zimeg) [inline packages/web-api/src/types/request/chat.ts:15]: This ordering seems unexpected to me - I'd've thought `@slack/types` finds a place before `./*` imports - but no blocker if this is what the linter likes 🙏 

Also only noticed this in a few `web-api` files... Not sure why though!

**Review by zimeg — APPROVED** [review]
Awesome PR 😭 The mental friction of "where do I put this import" shall no longer be. And this just looks a lot better. Thank you!

Left one comment with confusion on import ordering and one on comments, but neither are blocking! 🚀

File: packages/web-api/src/types/request/chat.ts (hunk 4 above)
Comment (filmaj) [reply packages/web-api/src/types/request/chat.ts:15]: I'm also not sure why.... but I almost don't want to ask questions. Just tell me how to order the damn things and don't make me think!

**Pull request merged**